nemo/collections/asr/models/rnnt_models.py

```diff
@@ -212,6 +212,7 @@ def transcribe(
         self,
         paths2audio_files: List[str],
         batch_size: int = 4,
+        return_encoder_embiddings: bool = False,
```

Comment:
Revert. Setup a parameter in the RNNTDecodingConfig and set this value there. 
We have to stop adding more arguments to model.transcribe(). Decoding framework exists for this purpose. Put the thing inside Hypothesis, and use return_hypotheses=True for model.transcribe().

Reply:
I moved `return_encoder_embiddings` parameter to `RNNTDecodingConfig`.